.dotnet.azure/src/Custom/FineTuning/AzureCreateJobOperation.Protocol.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Diagnostics.CodeAnalysis;
+
+#nullable enable
+
+namespace Azure.AI.OpenAI.FineTuning;
+
+/// <summary>
+/// A long-running operation for creating a new model from a given dataset.
+/// </summary>
+public class AzureCreateJobOperation : CreateJobOperation

```

Comment:
Curious about whether all of these LRO types need to be public ...

Reply:
I don't believe so -- I think these should be internal (and that, from the public API perspective, it'll just be another operation "implemented differently")

Reply:
in the case of create job operation the delete functions are not available on the base type so it must be public to access those. It would make sense that Azure would start to be a superset of functionalities and therefore add more here in the future.